core/Cargo.toml

```diff
@@ -53,7 +53,7 @@ clap = { version = "4.5.1", features = ["derive"], optional=true }
 realfft = "3.3.0"
 hashbrown = { version = "0.14.3", features = ["raw"] }
 scopeguard = "1.2.0"
-fluent-templates = "0.8.0"
+fluent-templates = { git = "https://github.com/XAMPPRocky/fluent-templates", branch = "master" }
```

Comment:
It would be nice to leave a comment here, linking to the PR that we need it for, and that the we can switch back to the next release after 0.8.0.